src/jit/gentree.h

```diff
@@ -1007,6 +1007,14 @@ struct GenTree
 
 #define GTF_ICON_SIMD_COUNT 0x04000000 // GT_CNS_INT -- constant is Vector<T>.Count
 
+#define GTF_ICON_INITCLASS 0x02000000 // GT_CNS_INT -- Constant is used to access a static that requires preceding
+                                      //               class/static init helper.  In some cases, the constant is
+                                      //               the address of the static field itself, and in other cases
+                                      //               there's an extra layer of indirection and it is the address
+                                      //               of the cell that the runtime will fill in with the address
+                                      //               of the static field; in both of those cases, the constant
+                                      //               is what gets flagged.
+
```

Comment:
Yeah, I noticed that.  I'll rebase/resolve if he merges first; I'd like to run desktop testing on this before merging.